tools/create-image.sh

```diff
@@ -17,6 +17,7 @@ sudo debootstrap --include=openssh-server,curl,tar,gcc,libc6-dev,time,strace,sud
 sudo sed -i '/^root/ { s/:x:/::/ }' $DIR/etc/passwd
 echo 'T0:23:respawn:/sbin/getty -L ttyS0 115200 vt100' | sudo tee -a $DIR/etc/inittab
 printf '\nauto eth0\niface eth0 inet dhcp\n' | sudo tee -a $DIR/etc/network/interfaces
+echo '/dev/root / ext4 defaults 0 0' | sudo tee -a $DIR/etc/fstab
```

Comment:
If it's the only change, does it also work for wheezy? Or otherwise do an if on RELEASE. It's kinda sucks to have a full copy-pasted version just for this. We can have default stretch, but if one wants to build wheezy they can override the RELEASE var in env, right?
syzbot uses a different script, and they already get out of sync from time to time:
https://github.com/google/syzkaller/blob/master/tools/create-gce-image.sh

Reply:
Okay, tested this with wheezy and it's fine, so I've just added the fstab line, updated the RELEASE, DIR, and updated docs now.